### PR TITLE
Added features

### PR DESCRIPTION
**Pull request by mehdibo**
- Deleting tags with backspace
- Max tags
- Animation for deleting

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,4 @@ $RECYCLE.BIN/
 
 # Mac crap
 .DS_Store
+SOTag.php
```
Comment (iog3) [inline .gitignore]: Why was this line added?

**Comment by mehdibo** [comment]
it was added automatically, when i said to ignore SOTag.php changes

**Comment by iog3** [comment]
Ah I see, if I merge your changes it will delete that file from the repository, if you are using github for windows / mac you can uncheck the sotag.php file so it won't get committed

**Comment by mehdibo** [comment]
no i don't think so, 
http://git-scm.com/docs/gitignore

**Comment by iog3** [comment]
@mehdibo I have included a few of your changes in the dev branch, I'll tag you in the pull request when its ready

**Comment by mehdibo** [comment]
Ok. Im new to github :p
